build.sbt

```diff
@@ -11,36 +13,38 @@ lazy val core = project
   .settings(commonSettings)
   .settings(
     name := "sbt-http4s-org",
-    addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "0.4.6"),
-    addSbtPlugin("com.codecommit" % "sbt-github-actions" % "0.9.4"),
+    addSbtPlugin("com.codecommit" % "sbt-spiewak" % "0.17.0"),
     addSbtPlugin("com.github.cb372" % "sbt-explicit-dependencies" % "0.2.15"),
     addSbtPlugin("com.lightbend.sbt" % "sbt-java-formatter" % "0.6.0"),
-    addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.8.1"),
-    addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.0"),
-    addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.6.0"),
     addSbtPlugin("io.chrisdavenport" % "sbt-mima-version-check" % "0.1.2"),
-    addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")
+    addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2"),
```

Comment:
We include this only so our workflow can depend on it, but repositories are still responsible for their own .scalafmt.conf. Is this why sbt-spiewak doesn't include it?